ci/recipe/meta.yaml

```diff
@@ -24,6 +24,8 @@ requirements:
     - qiime2 {{ release }}.*
     - q2-types {{ release }}.*
     - fasttree
+    - raxml
+    - iqtree
```

Comment:
Where is `iqtree` being used in this PR? Is it a dependency of `raxml`?

Reply:
Sorry, I misread your PR submission text above. How about we coordinate adding that dependency later, after we get this raxml method in?

Reply:
Okay will do. I'll remove the iqtree dependency.